core/iwasm/include/wasm_c_api.h

```diff
@@ -527,16 +527,20 @@ typedef struct WASMModuleCommon *wasm_module_t;
 #define LOAD_ARGS_OPTION_DEFINED
 typedef struct LoadArgs {
     char *name;
+    /* True by default.
```

Comment:
Could you change to be somewhat like `Used by wasm-c-api only, true by default`? Since LoadArgs is also used by wasm_runtime_load_ex, but this option doesn't take effect for it.